web/src/frontend/src/nav-soknad/faktum/InputFaktum.tsx

```diff
@@ -51,13 +53,15 @@ class InputFaktum extends React.Component<Props, State> {
 		this.setState(getStateFromProps(nextProps));
 	}
 
-	componentDidMount() {
-		this.props.dispatch(
-			registerFaktumValidering({
-				faktumKey: this.props.faktumKey,
-				rules: null
-			})
-		);
+	componentWillMount() {
+		if (this.props.required) {
+			this.props.dispatch(
+				registerFaktumValidering({
+					faktumKey: this.props.faktumKey,
+					valideringer: [...(this.props.required ? [validerRequired] : [])]
```

Comment:
Usikker på om de ulike formene for validering skal opprettes herfra basert på spesifikke props, eller om en skal sende inn array med valideringer som en egen prop... muligens enklere å sende inn tekster dersom det er en egen prop - validering som har mulighet for å sende inn feilmeldingstekst samtidig.
```
<InputFaktum
  {...}
  validering={
    [
      [validerRequired, "Feilmelding ved påkrevd"],
      [validerMinLengde, "Annen feilmelding"]
    ]  
  }/> 
```